tests/phpunit/includes/Core/Util/FakeUser_Input_Settings.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Class Google\Site_Kit\Tests\Core\Util\FakeUser_Input_Settings
+ *
+ * @package   Google\Site_Kit\Tests\Core\Storage
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Util;
+
+use Google\Site_Kit\Core\Util\User_Input_Settings;
+
+class FakeUser_Input_Settings extends User_Input_Settings {
```

Comment:
See my previous comments – let's mock this in the test instead. Since it's used in a few places we can add a method for creating it to not duplicate it.

Reply:
Updated.